bin/stacks/routing-api-stack.ts

```diff
@@ -189,7 +189,40 @@ export class RoutingAPIStack extends cdk.Stack {
     })
     quoteToRatio.addMethod('GET', routeToRatioLambdaIntegration)
 
-    const apiAlarm5xx = new aws_cloudwatch.Alarm(this, 'RoutingAPI-5XXAlarm', {
+    const apiAlarm5xxSev2 = new aws_cloudwatch.Alarm(this, 'RoutingAPI-SEV2-5XXAlarm', {
```

Comment:
Maybe we just leave comment that the comparison operator is default `GreaterThanOrEqualToThreshold`. Was just trying to figure out if the thresholds made sense for the diff severities but didnt know how the numbers were being compared